src/common/types/api.js

```diff
@@ -34,12 +57,34 @@ type ErrorContextProps = {
 
 type ElementsErrorCallback = (e: ElementsXhrError, code: string, contextInfo?: Object) => void;
 
+type XHROptions = {
+    apiHost?: string,
+    cache?: APICache,
+    clientName?: string,
+    consoleError?: boolean,
+    consoleLog?: boolean,
+    id?: string,
+    language?: string,
+    requestInterceptor?: Function,
+    responseInterceptor?: Function,
+    retryableStatusCodes?: Array<number>,
+    sharedLink?: string,
+    sharedLinkPassword?: string,
+    shouldRetry?: boolean,
+    token: Token,
+    uploadHost?: string,
+    version?: string,
+};
+
 export type {
+    Method,
```

Comment:
Can/should we alphabetize declarations and exports?

Reply:
The declarations have to be in order of usage to appease Eslint. In fact, I had to disable the rule on one line in types/core.js due to one type using the other and vice versa... no way to do that.

Do you prefer them in alpha with that rule disabled?

Reply:
That makes sense. I guess I'm thinking about named functions, where the declaration order doesn't matter. Feel free to leave them as-is.